### PR TITLE
Update docs theme

### PR DESCRIPTION
**Pull request by jordisala1991**
Dev-kit PR: https://github.com/sonata-project/dev-kit/pull/2448

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -91,7 +90,7 @@
 #show_authors = False
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = 'sphinx'
+pygments_style = 'default'
```
Comment (jordisala1991) [inline docs/conf.py]:
```suggestion
pygments_style = 'sphinx'
```

**Comment by jordisala1991** [comment]
WDYT @VincentLanglet .

This PR is not to merge as is, but I wanted to try latest versions of python deps.

Also I tried the latest rules for doc tor rst, They could be applied on another PR.

Things to change on all bundles is to remove the configuration-blocks used to show multiple formats of configurations. Thats used by symfony to show all possible formats, we mostly use yaml, just a few of them have yaml and xml, and I would say it is better to only show yaml, we dont have enough time to maintain multiple formats.

I will post an url to show the difference.

https://docs.sonata-project.org/projects/SonataAdminBundle/en/jordisala1991-patch-1/

Take a look at the code blocks, they should look better (and the attributes will also be printed nicer, you wont see them on AdminBundle tho, we dont use them yet)

**Comment by VincentLanglet** [comment]
> Things to change on all bundles is to remove the configuration-blocks used to show multiple formats of configurations. Thats used by symfony to show all possible formats, we mostly use yaml, just a few of them have yaml and xml, and I would say it is better to only show yaml, we dont have enough time to maintain multiple formats.

I agree

> I will post an url to show the difference.
> 
> [docs.sonata-project.org/projects/SonataAdminBundle/en/jordisala1991-patch-1](https://docs.sonata-project.org/projects/SonataAdminBundle/en/jordisala1991-patch-1/)
> 
> Take a look at the code blocks, they should look better (and the attributes will also be printed nicer, you wont see them on AdminBundle tho, we dont use them yet)

Seems better indeed

**Comment by jordisala1991** [comment]
This fixes both the problem with the build on readthedocs and the styles of code @VincentLanglet .

I will prepare a dev-kit PR. All repositories will need some fix in docs, it will be handled on the dev-kit PR itself.

Edit, done: https://github.com/sonata-project/dev-kit/pull/2448

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**